moveos/moveos-stdlib/moveos-stdlib/sources/context.move

```diff
@@ -65,11 +66,16 @@ module moveos_std::context {
         tx_context::fresh_address(&mut self.tx_context)
     }
 
-    /// Generate a new unique object ID
+    /// Generate a new unique ObjectID
     public fun fresh_object_id(self: &mut Context): ObjectID {
```

Comment:
Is this interface necessary because it destroys the life cycle of objectID and Object, and now they are not unified.
I can create ObjectID but not Object which feels weird

Reply:
I has removed it,but add it back.
Because 

1. Some places depend on it.
2. UID need to provide a public function to convert UID to ObjectID. 
3. Developers can pass arbitrary ObjectID by transaction arguments.